README.md

```diff
@@ -100,6 +128,10 @@ $("button").on("click", function(event){
 Your Answer:
 ```js
 // write code here
+var buttons = document.querySelectorAll("button");
+buttons.addEventListener("click", function() {
+  buttons.style.background = "red";
+});
 ```
```

Comment:
Very close. Because `querySelectorAll` returns a collection of DOM elements, you'll have to iterate through those using a loop and add the event listener inside of that loop.

It's a little different than jQuery and requires a little bit extra legwork.